src/caveexpress/server/entities/Platform.cpp

```diff
@@ -43,29 +43,17 @@ void Platform::onPreSolve (b2Contact* contact, IEntity* entity, const b2Manifold
 	b2WorldManifold worldManifold;
 	contact->GetWorldManifold(&worldManifold);
 	const b2Vec2 worldNormal = worldManifold.normal;
-	// not from above - then we don't care
-	// -1/sqrt(2)
-	if (worldNormal.y >= -0.707)
+	// not from above - then we don't care	// -1/sqrt(2)
+	if (worldNormal.y >= -0.07 ||
+		fabs(worldNormal.x) > fabs(worldNormal.y))
 		return;
 
 	const b2Manifold *maniFold = contact->GetManifold();
-	if (maniFold->pointCount < 2)
```

Comment:
here we should check that `pointCount` is `>= 1`- because we otherwise simple access a possible invalid array index a few lines below.

Reply:
Ah true, I didn't realize. Fixed.